diode-server/go.mod

```diff
@@ -20,6 +20,12 @@ require (
 )
 
 require (
+	github.com/alicebob/gopher-json v0.0.0-20200520072559-a9ecdc9d1d3a // indirect
+	github.com/yuin/gopher-lua v1.1.1 // indirect
+)
+
+require (
```

Comment:
Could we please keep these within one `require ()` block for indirects?

Reply:
done